Loop/Managers/TrustedTimeChecker.swift

```diff
@@ -45,7 +49,7 @@ class TrustedTimeChecker {
     }
     
     private func checkTrustedTime() {
-        ntpClient.fetchIfNeeded { [weak self] result in
+        ntpClient.fetchIfNeeded(completion: { [weak self] result in
```

Comment:
[nit] Why the addition of the completion argument? I don't believe it is needed.

Reply:
This is actually a new Xcode 12 compiler warning, believe it or not:
<img width="1542" alt="Screen Shot 2020-10-19 at 5 03 24 PM" src="https://user-images.githubusercontent.com/308993/96524406-0a442b00-122d-11eb-9101-3f7fc3f0cae6.png">